package.json

```diff
@@ -18,7 +18,7 @@
         "@elastic/eui": "101.3.0",
         "@emotion/css": "^11.11.2",
         "@emotion/react": "^11.11.1",
-        "@orchestrator-ui/orchestrator-ui-components": "*",
+        "@orchestrator-ui/orchestrator-ui-components": "^6.0.0",
```

Comment:
This means we won't auto upgrade anymore. You needed to pin this to fix the build errors?